tests/cdi/cdi_test.go

```diff
@@ -13,49 +13,39 @@ var _ = Describe("Importing and starting a VM using CDI", func() {
 	flag.Parse()
 	ktests.SkipIfNoOc()
 	dstPVCFilePath := "/tmp/test-pvc.json"
-	dstPVCFilePath1 := "/tmp/test-pvc1.json"
 	dstVMFilePath := "/tmp/test-vm.json"
 
 	url, ok := os.LookupEnv("STREAM_IMAGE_URL")
 	if !ok {
 		url = pvcEPHTTPNOAUTHURL
 	}
 
-	Context("PVC with valid image url will succeed", func() {
-		It("should create the PVC successfully", func() {
-			tests.ProcessTemplateWithParameters(rawPVCFilePath, dstPVCFilePath, "PVC_NAME="+pvcName, "EP_URL="+url)
-			tests.CreateResourceWithFilePathTestNamespace(dstPVCFilePath)
-		})
+	BeforeEach(func() {
+		tests.ProcessTemplateWithParameters(rawPVCFilePath, dstPVCFilePath, "PVC_NAME="+pvcName, "EP_URL="+url)
+	})
```

Comment:
The `tests.ProcessTemplateWithParameters` runs here and invokes in the 2nd context again? I think it does not make sense, how about put it into `JustBeforeEach`?

Reply:
updated